test cases/windows/14 vs c++17/meson.build

```diff
@@ -0,0 +1,8 @@
+project('wincpp', 'cpp')
+
+if meson.get_compiler('cpp').version().version_compare('<19.11')
+    error('MESON_SKIP_TEST Visual Studio 2017 (version 15.3) or later is required for C++17 support')
```

Comment:
Why are you skipping the test here? Also where's the `cpp_std:` option set for this test?